### PR TITLE
[UI/UX:InstructorUI] Limit Course Settings Textbox Resize

### PR DESCRIPTION
**Pull request by itssandeepverma**
User resizing markdown text area #10008 

 ### What is the current behavior?
Textboxes that can be resized by the user can be expanded as much or as little as possible. I assumed it's supposed to stay fixed with auto resizing or the user is only suppose to be able to resize vertically.

See also #10016 for what it looked like before

### What is the new behavior?
Resize it vertically to be consistent with SQL-Toolbox change, #10016.

https://github.com/Submitty/Submitty/assets/82904790/8215fa6f-66e7-44aa-a7a1-646843cef302

### Other information?

Checked by running on remote server.

### CODE REVIEW

**Review by ArnavPawar — CHANGES_REQUESTED** [review]
This fix already has been made it is a duplicated PR. I will attach the PR that is associated with this in a bit.

**Comment by itssandeepverma** [comment]
> This fix already has been made it is a duplicated PR. I will attach the PR that is associated with this in a bit.

Kindly mention it. Thanks

File: site/app/templates/admin/Configuration.twig [hunk 1]
```diff
@@ -168,8 +168,8 @@
                 What instructions should be shown to students before they enter a message?
             </span>
         </label>
-        <textarea class="config-row" name="seek_message_instructions" id="seek-message-instructions">{{ fields['seek_message_instructions'] }}</textarea>
-
+        <textarea class="config-row" name="seek_message_instructions" id="seek-message-instructions" style="resize:vertical; height: 290px;">{{ fields['seek_message_instructions'] }}</textarea>
+
```
Comment (04vmatsibekker) [inline site/app/templates/admin/Configuration.twig:172]: remove space here, and put styling into a css file

File: site/app/templates/admin/Configuration.twig [hunk 2]
```diff
@@ -201,14 +201,14 @@
             <br>
             <span class="option-alt">Input the message shown on grades returned to students.  Include instructions for grade inquiries and the course staff mailing list.</span>
         </label>
-        <textarea class="config-row" name="course_email" id="course-email">{{ fields['course_email'] }}</textarea>
+        <textarea class="config-row" name="course_email" id="course-email" style = "resize:vertical ;">{{ fields['course_email'] }}</textarea>
```
Comment (04vmatsibekker) [inline site/app/templates/admin/Configuration.twig:204]: put styling into css file

**Review by 04vmatsibekker — CHANGES_REQUESTED** [review]

**Review by zacharymnp — APPROVED** [review]

**Pull request merged**